### PR TITLE
Blog app - integration specs for views

### PR DESCRIPTION
**Pull request by JbirdL86**
what's new :question: 

- Set capybara testing configurations
- Add tests for login page
- Add tests for user index page
- Add tests for user show page
- Add tests for post index page
- Add tests for post show page

### CODE REVIEW

File: spec/system/post_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          recent_posts.each do |post|
+            click_link post.name
+            expect(current_path).to eq(user_post_path(user.id, post.id))
+          end
+          visit root_path
+        end
+      end
+    end
+  end
+end
```
Comment (Emmyn5600) [inline spec/system/post_index_spec.rb]: - [ ] Let us add the remaining tests for `User` **post index page**:

* Tests that assert you can see the user's profile picture.
* Tests that assert you can see the user's username.
* Tests that assert you can see the number of posts the user has written.
* Tests that assert you can see a post's title.
* Tests that assert you can see some of the post's body.
* Tests that assert you can see the first comments on a post.
* Tests that assert you can see how many comments a post has.
* Tests that assert you can see how many likes a post has.
* Tests that assert you can see a section for pagination if there are more posts than fit on the view.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @Laylo309, @nelsinofrancisco   and @JbirdL86 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.

Highlights:

- The project is running successfully
- You have a descriptive Readme file
- You have implemented almost all requirements.
- All tests are passing successfully.
- You have descriptive PR
      
 ## Required Changes ♻️

- Check the comment under the review

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: spec/system/post_index_spec.rb (hunk 1 above)
Comment (Laylo309) [reply spec/system/post_index_spec.rb]: Everything is done✅✅✅

**Review by descholar-ceo — APPROVED** [review]
Hi @JbirdL86  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Laylo309** [comment]
> Hi @JbirdL86 ,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hello I cannot see submit button in my dashboard

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @Laylo309,

Good job on implementing all the requirementss. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**